### PR TITLE
CSP-360 extra endpoint added to external for provider course details

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.Roatp.Application.UnitTests/ProviderCourse/Queries/ProviderCourseQueryHandlerTests.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@
 namespace SFA.DAS.Roatp.Application.UnitTests.ProviderCourse.Queries
 {
     [TestFixture]
-    public class ProviderCourseQueryHandlerTests
+    public class ProviderCourseCourseQueryHandlerTests
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.Application.UnitTests/ProviderCourse/Queries/ProviderCourseQueryHandlerTests.cs:14]: Don't think this is deliberate?

File: src/SFA.DAS.Roatp.Application.UnitTests/Providers/Queries/GetCourse/CourseQueryHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+using NUnit.Framework;
+using SFA.DAS.Roatp.Application.ProviderCourse.Queries.GetProviderCourse;
+using SFA.DAS.Roatp.Domain.Entities;
+using SFA.DAS.Roatp.Domain.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.Roatp.Application.UnitTests.Providers.Queries.GetCourse
+{
+    [TestFixture]
+    public class CourseQueryHandlerTests
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.Application.UnitTests/Providers/Queries/GetCourse/CourseQueryHandlerTests.cs:14]: ProviderCourseQueryHandlerTests

File: src/SFA.DAS.Roatp.Application/ProviderCourse/Queries/GetProviderCourse/GetCourseQueryHandler.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    private readonly ILogger<GetProviderCourseQueryHandler> _logger;
+
+    public GetCourseQueryHandler(IProviderCoursesReadRepository providerCoursesReadRepository, IStandardsReadRepository standardsReadRepository, ILogger<GetProviderCourseQueryHandler> logger)
+    {
+        _providerCoursesReadRepository = providerCoursesReadRepository;
+        _standardsReadRepository = standardsReadRepository;
+        _logger = logger;
+    }
+
+    public async Task<GetCourseQueryResult> Handle(GetCourseQuery request, CancellationToken cancellationToken)
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.Application/ProviderCourse/Queries/GetProviderCourse/GetCourseQueryHandler.cs:23]: Cannot we use existing query and handler?

File: src/SFA.DAS.Roatp.Application.UnitTests/Providers/Queries/GetCourse/CourseQueryHandlerTests.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Roatp.Application.ProviderCourse.Queries.GetProviderCourse;
+using SFA.DAS.Roatp.Domain.Entities;
+using SFA.DAS.Roatp.Domain.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.Roatp.Application.UnitTests.Providers.Queries.GetCourse
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.Application.UnitTests/Providers/Queries/GetCourse/CourseQueryHandlerTests.cs:11]: I am confused with names space, shouldn't this be in  `Providers.Queries.GetProviderCourse`

**Review by DevenKShah — COMMENTED** [review]
I have some questions.

File: src/SFA.DAS.Roatp.Application.UnitTests/ProviderCourse/Queries/ProviderCourseQueryHandlerTests.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.Roatp.Application.UnitTests/ProviderCourse/Queries/ProviderCourseQueryHandlerTests.cs:14]: no,typo, will update

File: src/SFA.DAS.Roatp.Application.UnitTests/Providers/Queries/GetCourse/CourseQueryHandlerTests.cs (hunk 4 above)
Comment (MarkFCain) [reply src/SFA.DAS.Roatp.Application.UnitTests/Providers/Queries/GetCourse/CourseQueryHandlerTests.cs:11]: I'll look at this

File: src/SFA.DAS.Roatp.Application/ProviderCourse/Queries/GetProviderCourse/GetCourseQueryHandler.cs (hunk 3 above)
Comment (MarkFCain) [reply src/SFA.DAS.Roatp.Application/ProviderCourse/Queries/GetProviderCourse/GetCourseQueryHandler.cs:23]: Here's what sonarcloud complained about: https://github.com/SkillsFundingAgency/das-roatp-api/runs/9639309767

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**